src/ORM/EagerLoadable.php

```diff
@@ -86,6 +86,14 @@ class EagerLoadable
     protected $_forMatching;
 
     /**
+     * The property name where the association result should be nested
+     * in the result.
+     *
+     * @var string
+     */
+    protected $_targetProperty;
```

Comment:
`propertyPath` is the way up to the container of `targetProperty`. For example

```php
$article->author->profile->company;
```

Would have a `propertyPath` of `author.profile` and a `targetProperty` of `company`